@here/olp-sdk-dataservice-api/lib/lookup-api.ts

```diff
@@ -106,7 +106,9 @@ export async function platformAPI(
  *
  * @summary Return the list of the platform APIs.
  */
-export async function platformAPIList(builder: RequestBuilder): Promise<API[] | ApiNotFoundError> {
```

Comment:
Please use deprecation instead of changes to avoid API breaks

Reply:
Done